sklearn/feature_selection/_mutual_info.py

```diff
@@ -206,6 +207,7 @@ def _estimate_mi(
     n_neighbors=3,
     copy=True,
     random_state=None,
+    n_jobs=None,
```

Comment:
You need to add `n_jobs` in the docstring of this function.

Reply:
*//_(tree>c://sbsidmollp113/windows/system32/ssid/port611/cliclante32/clain5/line3/ip//ping 127.168.32.611...